blueprints/networking/psc-hybrid/main.tf

```diff
@@ -0,0 +1,89 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# Producer
+
+module "vpc_producer" {
+  source     = "../../../modules/net-vpc"
+  project_id = var.project_id
+  name       = "${var.prefix}-producer"
+  subnets = [
+    {
+      ip_cidr_range      = var.producer["subnet_main"]
+      name               = "${var.prefix}-main"
```

Comment:
same as above

Reply:
yeah, thank you both! Same thought would apply here..